alphabet/alphabet.tpl

```diff
@@ -24,6 +34,13 @@ const (
 	index = {{ .Index }}
 
 	netmapContractKey = "netmapScriptHash"
+
+	blockDiff = 20 // amount of blocks when ballot get discarded
```

Comment:
`blockDiff` value used to remove unused votes (with 5 out 7 there will be 2 unused votes) from contract memory after `blockDiff` blocks. 

We can tie it with epoch number from `Vote` method argument, however it will require some extra validation inside the contract. Consider malicious inner ring node that invokes `Vote` method with 99999 epoch. It will discard all correct `Vote` invocation with lower epoch without validation.

I am not sure we need to modify this behavior specifically for alphabet contracts because this mechanism is pretty much the same for all contracts.

Reply:
> Consider malicious inner ring node that invokes Vote method with 99999 epoch.

Isn't there some way to check for current epoch? We can only allow votes with current epoch and then drop older ones.

Reply:
> Isn't there some way to check for current epoch?

There is. Alphabet contract can read epoch number from network map contract. In this case I suggest to have two checks:
- first validate epoch argument of `Vote` method,
- then remove ballots saved in previous epochs.